src/modules/roc_core/format_tid.h

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2019 Roc authors
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+//! @file roc_core/format_tid.h
+//! @brief Retrieve the current thread.
+
+#ifndef ROC_CORE_FORMAT_TID_H_
+#define ROC_CORE_FORMAT_TID_H_
+
+#include <sys/types.h>
```

Comment:
`sys/types.h` header is not portable. Since this file (src/modules/roc_core/format_tid.h) does not belong to a target directory, we should include only portable headers here. I guess we can just remove this one.